src/equations/compressible_euler_multicomponent_1d.jl

```diff
@@ -393,27 +449,25 @@ end
 
   rho       = density(u, equations)
 
```

Comment:
I'm not an expert on the multicomponent stuff - could you please explain the rational behind this change briefly? Was is a bug fix? Or is something else going on?

Reply:
You mean the cons2entropy change right? It's actually the same stuff but written in a much cleaner (and, with regard to the singlecomponent case, more similar ) way.

Reply:
No, I was talking about the changes in `max_abs_speed_naive` below

Reply:
Sorry, you're right, I mixed up the lines